samcli/lib/list/resources/resource_mapping_producer.py

```diff
@@ -0,0 +1,80 @@
+"""
+in progress
+"""
+from typing import Optional, Any
+import dataclasses
+import logging
+import yaml
+import click
```

Comment:
same comment as the other one on not using click outside of `commands`

Reply:
Fixed